src/types.ts

```diff
@@ -104,23 +113,29 @@ export type JoinByCase<T, Joiner extends string> = string extends T
         ? JoinLowercaseWords<T, Joiner>
         : never;
 
-export type PascalCase<T> = string extends T
+export type PascalCase<
+  T,
+  Normalize extends boolean | undefined,
```

Comment:
This is a breaking type change as it now requires a second generic. Is that intended?